templates/_helpers.tpl

```diff
@@ -148,6 +148,8 @@ go:
 {{- end }}
 
 {{- if stencil.Arg "kubernetes.groups" }}
+- name: github.com/getoutreach/k8slib
+  version: v1.0.0
```

Comment:
question: @evadnoob this (use of `kubernetes.groups` in `service.yaml`) is the only condition in `stencil-golang` where `k8slib` needs to be included, correct?

Reply:
yes, I think so. I'll update to match the same versions in go.mod for k8slib